src/System.Private.ServiceModel/src/project.json

```diff
@@ -17,7 +17,6 @@
     "System.Linq.Expressions": "4.0.10",
     "System.Linq.Queryable": "4.0.0",
     "System.Net.Http": "4.0.0",
-    "System.Net.Primitives": "4.0.10",
     "System.Net.WebHeaderCollection": "4.0.0",
```

Comment:
Question only: does this removal reflect the fact we never needed this?  Or does the need to away due to the later NuGet packages?